webplugin/js/app/km-post-initialization.js

```diff
@@ -7,13 +7,17 @@
 Kommunicate.postPluginInitialization = function (err, data) {
     // get the third party settings
     KommunicateKB.init(Kommunicate.getBaseUrl());
-    Kommunicate.getFaqList(data);
+    var categoryName;
+    if (kommunicate && kommunicate._globals && kommunicate._globals.getFaqByCategory) {
```

Comment:
@devashishmamgain is this parameter name is okay `getFaqByCategory` ?

Reply:
no, this looks like a function name

Reply:
@Prochnost55, please check this and update parameter name accordingly

Reply:
ok i have changed it to faqCategory